langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/CollectionCreationRequest.java

```diff
@@ -0,0 +1,50 @@
+package dev.langchain4j.store.embedding.chroma;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class CollectionCreationRequest {
+
+    private final String name;
+    private final Map<String, String> metadata;
+
+    /**
+     * Currently, cosine distance is always used as the distance method for chroma implementation
```

Comment:
please move/copy this comment into ChromaEmbeddingStore

Reply:
added comment to ChromaEmbeddingStore